tests/test_pytest_randomly.py

```diff
@@ -10,6 +10,10 @@
 pytest_plugins = ["pytester"]
 
 
+# See https://github.com/pytest-dev/pytest-randomly/issues/539
+LINE_START = 9 if pytest.version_tuple >= (7, 3) else 8
```

Comment:
Thanks for proving that this is the only change needed, but I'm fine with leaving the tests only passing on one pytest version. We don't test multiple versions on CI, so there's no point in complicating the test code.

Reply:
I assumed other distros might want the tests to work with older pytest as well. But I know supporting that is not your obligation. :balance_scale:

Reply:
Yeah, I don't support running the tests outside of this repo's tox setup. It has only led to false positives and busywork in the past...

Reply:
This PR was motivated by problems discovered by running the tests of this repo in the Fedora RPM package, do you consider that busywork?

Reply:
I would have eventually upgraded pytest in this repo (I upgrade the pinned deps every two weeks) so would have found the issue then. That you found it earlier, great. So not busywork.